### PR TITLE
Host existing iOS SDK documentation.

### PR DESCRIPTION
**Pull request by drn**
Moving SDK documentation from internal wiki to SDK repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,294 @@
...
+
+The SDK will be delivered as a native Swift library distributed via the package
+manager Cocoapods.
+
+## Authentication
+
+### Merchant
+
+Thanx will provide the SDK `Client ID` and a `Client Secret` via a secure
+channel.
```
Comment (colesteaks) [inline README.md:16]: would it be worth adding this to the dashboard somewhere?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,294 @@
...
+# Present it as a full screen modal
+let thanxWebViewController = Thanx.WebViewController(showToolbar: true)
+present(thanxWebViewController, animated: true)
+
+# Present it as a part of a UINavigationController
+let thanxWebViewController = Thanx.WebViewController(showToolbar: false)
+navigationController?.pushViewcontroller(thanxWebViewController, animated: true)
+```
+
+## Push Notifications
```
Comment (colesteaks) [inline README.md:153]: are we sure these work?

**Review by colesteaks — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (drn) [reply README.md:16]: For now, we'll be exchanging credentials with them via an SFTP server

File: README.md (hunk 2 above)
Comment (drn) [reply README.md:153]: Yup - just tested with Lou's app using the SDK:
img_2961

**Pull request merged**